### DISCUSSION TITLE
Command for readme on repos

### DISCUSSION THREAD

**Discussion by superduppperme** [nodeschool/discussions#2951] [Q&A] [answered]
Everytime I create a new repository, I see these instructions:

![image](https://github.com/nodeschool/discussions/assets/159910556/39430225-a7e5-4e73-ad6f-4b18fceafa21)

Instead  of using `echo "# nvm-extension" >> README.md`, wouldn't it be correct to use `echo "# nvm-extension" > README.md` instead since there is no README.md in this new repository?

**Comment by Gumaan-Mahar** [comment, marked as answer]
Yes, you are correct. Both commands achieve the same outcome in this context, which is creating a new file named README.md with the content "# nvm-extension". The difference lies in how they handle the file creation:

1. `echo "# nvm-extension" >> README.md`: This appends the text "# nvm-extension" to an existing README.md file if it exists. If the file doesn't exist, it creates a new one and adds the text to it.

2. `echo "# nvm-extension" > README.md`: This command writes the text "# nvm-extension" to a new file named README.md. If README.md already exists, it overwrites its content with "# nvm-extension".

So, in the context of creating a new repository where README.md doesn't exist yet, using `>` to create the file is more appropriate and efficient, as it ensures that the file is created with the desired content without any unnecessary append operations.

**Comment by superduppperme** [comment]
ok, thanks @Gumaan-Mahar
